index.html

```diff
@@ -35,29 +35,43 @@ <h2>Question:</h2>
     </main>
     <nav class="app-grid__footer">
       <ul role="list" class="navigation">
-        <li>
+        <li class="navigation__button navigation__button--active">
```

Comment:
I am a little confused but on the other hand i did not start yet with the navigation. The active modification should apply to each button and then only when it is active right? dont you need to apply this on all buttons, so in `<ul>`?

Reply:
I actually just want to change the color of the button of the active page. So if you are on the index.html I want to have the home button in another color :)